### PR TITLE
add Transifex in README

### PR DESCRIPTION
**Pull request by lemelon**

### CODE REVIEW

**Comment by frankrousseau** [comment]
@lemelon can you rebase this PR?

File: .tx/config [hunk 1]
```diff
@@ -5,5 +5,4 @@ host = https://www.transifex.com
 file_filter = client/app/locales/<lang>.json
 source_file = client/app/locales/en.json
 source_lang = en
-trans.fr = client/app/locales/fr.json
```
Comment (m4dz) [inline .tx/config]: Why suppressing `fr` locale here?

**Comment by frankrousseau** [comment]
@m4dz can you handle the fix and include this PR in development?

**Comment by m4dz** [comment]
Yep, I'll do it :+1:

**Comment by m4dz** [comment]
Changes finally pulled in #152. Thanks @lemelon :smiley:

**Pull request closed without merging**